BookingManager-persistence/src/main/java/cz/muni/fi/pa165/brown/entity/Reservation.java

```diff
@@ -0,0 +1,157 @@
+package cz.muni.fi.pa165.brown.entity;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+/**
+ * Class for Reservation entity
+ *
+ * @author Peter Hutta
+ * @version 1.0  25.10.2016
+ */
+@Entity
+@Table(name = "Reservations")
+public class Reservation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    @ManyToOne
+    private User user;
+
+    @NotNull
+    @ManyToOne
+    private Room room;
+
+    @NotNull
+    private Date from;
```

Comment:
Please change the name of this field to something which does not match sql keyword exactly (e.g. reservedFrom). While I was implementing the tests I found out that "from" field name caused trouble in SQL statements that hibernate produces, which has caused the `org.hibernate.exception.SQLGrammarException`